packages/teams-js/src/private/externalAppAuthentication.ts

```diff
@@ -120,7 +168,7 @@ export namespace externalAppAuthentication {
 
   /**
```

Comment:
On line 152 (in the new file line numbers), we have `composeExtension?: ComposeExtensionResponse` as an optional parameter. Did we get confirmation that it is indeed optional or can we make this required?